pure_pursuit_follower:
  planning_time: 1.0
  min_lookahead_distance: 6.0
  wheel_base: 2.789
  heading_angle_limit: 90.0
  lateral_error_limit: 2.0
  nearest_neighbor_search:  kd_tree  # kd_tree. ball_tree, brute, auto
  publish_debug_info: False

stanley_follower:
  wheel_base: 2.789
  cte_gain: 0.4     # gain for cross track error
  heading_angle_limit: 90.0
  lateral_error_limit: 2.0
  nearest_neighbor_search: kd_tree  # kd_tree. ball_tree, brute, auto
  publish_debug_info: False
